client/media.js

```diff
@@ -34,6 +54,42 @@ const stopStreams = () => {
     });
 };
 
+const getFormatDate = (date) => {
+    return date.toISOString().replace(/\.[^.]*$/, '');
+};
+
+const getDifferenceInTime = (date1, date2) => {
+    const diff = Math.abs(date2.getTime() - date1.getTime()); // ms
+    const seconds = Math.round(diff / 1000);
+    const minutes = Math.round(seconds / 60);
+    const hours = Math.round(minutes / 60);
```

Comment:
К [предыдущему замечанию коллеги](https://github.com/moevm/mse1h2025-prctr/pull/55#pullrequestreview-2702036905) добавлю.

Неправильно вычислены секунды, минуты и часы. Сначала вычисляется количество целых часов, через разность вычисляется количество целых минут после чего через разность вычисляется количество целых секунд.

```suggestion
    const totalSeconds = Math.floor(diff / 1000);
    const hours = Math.floor(totalSeconds / 3600);
    const minutes = Math.floor((totalSeconds % 3600) / 60);
    const seconds = totalSeconds % 60;

    // Для удобного представления
    const formattedHours = String(hours).padStart(2, '0');
    const formattedMinutes = String(minutes).padStart(2, '0');
    const formattedSeconds = String(seconds).padStart(2, '0');

    return `${formattedHours}:${formattedMinutes}:${formattedSeconds}`;
```